### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by CindyDorantes**
First review request included:
- Add desktop version styling for sections headline, works, about me, and contact, using media query.

Second review request. Changes included:
- update nav element for the desktop version in headline section
- fix space between headline section and recent works title

### CODE REVIEW

File: Index.html [hunk 1]
```diff
@@ -14,38 +14,36 @@
 <body>
     <header class="toolbar">
         <a class="logo">Welcome</a>
-        <nav class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></nav>
-        
+        <nav class="hamburguer"><img src="./Images/Union.png" alt="hamburguer menu"></nav>        
     </header>
```
Comment (roshan-bajgain) [inline Index.html:18]: - [ ] The Item Navigation should be at the center. Kindly fix it as you don't have a navigation Item on the desktop screen.
### Figma design
figma

File: Index.html [hunk 2]
```diff
@@ -14,38 +14,36 @@
...
             If you like what you see and have a project you need coded, don't hesitate to contact me.
         </p>
-        
-        <ul>
+        <ul class="social-heading">
             <li><a href="https://twitter.com/CindyDorantes10"><img src="./Images/twitter.png" alt="Twitter icon" class="socialicon"></a></li>
             <li><a href="https://www.linkedin.com/in/cindydorantessanchez/"><img src="./Images/Linkedin_icon.png" alt="LinkedIn icon" class="socialicon"></a></li>
             <li><a href="https://medium.com/"><img src="./Images/medium.png" alt="Medium icon" class="socialicon"></a></li>
             <li><a href="https://github.com/CindyDorantes"><img src="./Images/github.png" alt="GitHub icon" class="socialicon"></a></li>
             <li><a href="https://angel.co/"><img src="./Images/angellist.png" alt="AngelList icon" class="socialicon"></a></li>
```
Comment (roshan-bajgain) [inline Index.html:37]: - [ ] After the first section, there is more spacing on the button so kindly increase the spacing there: Here is the figure:
### your design
your design
### Figma design
figma

**Review by roshan-bajgain — CHANGES_REQUESTED** [review]
Hi @CindyDorantes, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Professional README file. ✔️
- PR has a good title and summary. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OmarRamoun — APPROVED** [review]
# 🟢 Project Approved! 🏆

## Hi @CindyDorantes 👋

Your project is complete! ✔️ 🏁

Finally, **you did it**! 🎉 👏. Your code was very professional 💯 and I can see you have a great chance to be a Great Software Engineer ⛑️.

There is nothing else to say other than... _it's time to merge it_ :shipit:
Congratulations! 🎉. And Welcome to Microverse 👋.

> **Note:** You can now take a break even if short before you move on to the next project. Happy 😄 Coding 🖥️! and Have a nice day 🌞.

## Highlights 🌟

- ✔️ Descriptive commmit messsages
- ✔️ Descriptive PR description
- ✔️ No linter Errors
- The code is top-notch! 💯 and have a huge potential.

Cheers and Happy coding!👏👏👏

## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [@omarramoun](https://github.com/omarramoun)
- This is My **Twitter**: [@omarramoun](https://twitter.com/omarramoun)
- This is My **Slack**: [Ramoun](https://slack.com)
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**